pde2e-image/lib/windows/runner.ps1

```
@@ -1,18 +1,22 @@
 param(
-    [Parameter(HelpMessage='url to download the exe for podman desktop, in case we want to test an specific build')]
+    [Parameter(HelpMessage='url to download the exe for app to test, in case we want to test an specific build')]
     $pdUrl="",
-    [Parameter(HelpMessage='path for the exe for podman desktop to be tested')]
+    [Parameter(HelpMessage='path to the executable of the application to be tested')]
     $pdPath="",
     [Parameter(Mandatory,HelpMessage='folder on target host where assets are copied')]
     $targetFolder,
     [Parameter(Mandatory,HelpMessage='Results folder')]
     $resultsFolder="results",
```

PSScriptAnalyzer (1.25.0)

[warning] Mandatory Parameter 'resultsFolder' is initialized in the Param block. To fix a violation of this rule, please leave it uninitialized.

(PSAvoidDefaultValueForMandatoryParameter)